orderbook/src/orderbook.rs

```diff
@@ -117,7 +120,10 @@ impl Orderbook {
                 order.sell_token_balance,
             ));
         }
-        if order.signature.scheme() == SigningScheme::PreSign {
+        if !matches!(
```

Comment:
> That being said, I haven't put much thought into it. Do you have a good use case for excluding all "presign" orders if the flag is turned off?

My thought was that if we get spammed we will accrue some orders in the database before turning on this flag. But we can also manually remove them in that case (if they are already overwhelming the solver).

Reply:
> if they are already overwhelming the solver

The spam orders shouldn't get to the solver as they would get filtered out in the solvable orders query.

With that being said, I guess it could be possible that the orderbook API would get overwhelmed by the spam causing the orderbook query to take too long - but, intuitively, the solvable order query would still have to run the entire order table even if we filter out all `signing_scheme = "presign"` orders, so my assumption would be that it wouldn't speed up the query that much (assuming that the `LEFT JOIN` that we are doing is performant, which it hopefully is, as we index the latest pre-signature event per order UID). If we additionally index the `signing_scheme` column, then it might make a more significant difference, but then we would have to maintain multiple "query paths" depending on whether or not the flag is enabled which I would like to avoid. **Disclaimer**, a lot of this is based on speculation as I don't really understand Postgres.

Reply:
Sounds good. Let's merge it like this!